src/commands/push.ts

```diff
@@ -74,10 +74,14 @@ export default class Push extends Command {
         });
         cli.action.stop(chalk.green(`Image verified`));
 
+        const config = classToPlain(node.service_config); // debug block data for local should not be posted
+        if ((config as any).debug) {
+          (config as any).debug.path = undefined;
```

Comment:
Why do we need to set path to undefined? I thought validation would prevent this.